ui-v2/tests/integration/services/repository/coordinate-test.js

```diff
@@ -44,3 +44,18 @@ test('findAllByDatacenter returns the correct data for list endpoint', function(
     }
   );
 });
+test('findAllByNode calls findAllByDatacenter with the correct arguments', function(assert) {
+  assert.expect(3);
+  const datacenter = 'dc-1';
+  const conf = {
+    cursor: 1,
+  };
+  const service = this.subject();
+  service.findAllByDatacenter = function(dc, configuration) {
+    assert.equal(arguments.length, 2, 'Expected to be called with the correct number of arguments');
+    assert.equal(dc, datacenter);
+    assert.deepEqual(configuration, conf);
```

Comment:
`configuration` is not set here so it equals {}. How does it equal `cursor: 1`? Should it not equal `index:1`?

Reply:
Remember we are completely replacing the `findAllByDatacenter` method here. `findByNode` calls our fake `findAllByDatacenter` with the `dc` and `configuration` arguments, which it gets from the original call to `findAllbyNode('node-name', datacenter, conf)`, `conf` being `{cursor: 1}`, so eventually `configuration === {cursor: 1}`

Previously we were expecting and passing through an extra `nspace` argument, which was actually the `configuration` object, which was the root of the bug.